core/domain/feedback_services.py

```diff
@@ -261,3 +262,27 @@ def enqueue_feedback_message_email_task(user_id):
     taskqueue_services.enqueue_task(
         feconf.FEEDBACK_MESSAGE_EMAIL_HANDLER_URL, {'user_id': user_id},
         feconf.DEFAULT_FEEDBACK_MESSAGE_EMAIL_COUNTDOWN_SECS)
+
+
+def send_feedback_message_email(exploration_id, thread_id, message_id):
+    """Creates"""
+    exploration_rights = rights_manager.get_exploration_rights(exploration_id)
+    feedback_message = {
```

Comment:
Let's call this a feedback_message_reference, and change the storage model property to feedback_message_references instead of feedback_messages.

Sorry I'm only realizing this now -- this came up in another discussion where I was doing something similar in a PR, and we realized that using *_references was a good way to differentiate this sort of thing from the feedback messages themselves.